tests/unit/model/model_test.py

```diff
@@ -7,7 +8,17 @@
 from rflx.error import Location, RecordFluxError
 from rflx.expression import Number
 from rflx.identifier import ID
-from rflx.model import BUILTIN_TYPES, Enumeration, Model, ModularInteger, RangeInteger, Type
+from rflx.model import (
+    BUILTIN_TYPES,
+    Enumeration,
+    Message,
+    Model,
+    ModularInteger,
+    RangeInteger,
+    Type,
+)
+from rflx.model.message import FINAL, INITIAL, Field, Link
+from rflx.model.type_ import Sequence as MSequence
```

Comment:
To make this code consistent to the existing code, I would suggest to use `from rflx.model import type_ as mty` and `mty.Sequence` instead of `MSequence`.